internal/tfengine/tfengine.go

```diff
@@ -91,10 +92,19 @@ func Run(confPath, outPath string, opts *Options) error {
 		}
 	}
 
-	if err := os.MkdirAll(outPath, 0755); err != nil {
-		return fmt.Errorf("failed to mkdir %q: %v", outPath, err)
+	if _, err := os.Stat(outPath); os.IsNotExist(err) {
+		if err := os.MkdirAll(outPath, 0755); err != nil {
+			return fmt.Errorf("failed to mkdir %q: %v", outPath, err)
+		}
+	} else {
+		diffs, err := fileutil.DiffDirs(outPath, tmpDir)
+		if err != nil {
+			return fmt.Errorf("failed to find diff between dir: %v", err)
+		}
+		if len(diffs) > 0 {
+			log.Printf("Unmanaged files:\n%v", strings.Join(diffs, "\n"))
```

Comment:
just so i understand, for common files, it will still be an override right?

Reply:
Yeah, this doesn't actually change any behaviour. It just prints files that were not generated by this run, so in case the user changes the template there aren't old files that are missed.